dev/test-zeekoe.py

```diff
@@ -263,7 +265,7 @@ def run_command_list(self, command_list):
 def main():
     parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter)
     parser.add_argument("command", help="", nargs="?", default="list")
-    parser.add_argument("--path", help="path to create configs", default=".")
+    parser.add_argument("--config-path", help="path to create configs", default=".")
```

Comment:
The usage also needs to be updated on L292

Reply:
oh yes, good catch. I've renamed `dev_path` to `config_path` everywhere else too.